sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusReceiverClient.cs

```diff
@@ -81,240 +81,120 @@ public class ServiceBusReceiverClient : IAsyncDisposable
         ///   and should take responsibility for managing its lifespan.
         /// </summary>
         ///
-        internal bool OwnsConnection { get; set; } = true;
+        internal bool OwnsConnection { get; set; } = false;
```

Comment:
Was this change intentional?   I see it getting flipped to `false` for the protected mocking constructor, but I don't see it set `true` elsewhere.

Reply:
Yeah based on the Api Updates PR, this will always be false. It can probably be removed entirely.